sdcm/nemesis.py

```diff
@@ -2800,7 +2800,7 @@ def is_streaming_started():
             return len(streaming_logs) > 0 or len(repair_logs) > 0
 
         wait.wait_for(func=is_streaming_started, timeout=600, step=1,
```

Comment:
AFAIU the issue is that, this timeout is not being enforced, right?

Reply:
the issue that if the timeout passed, nothing happens, and we continue to the next step